content/projects/delfin/user-guide/dashboard/_index.md

```diff
@@ -12,9 +12,10 @@ delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open
 It provides unified APIs to access, export and connect with clients as well as a set of interfaces for various driver additions. These APIs combined with the SODA Dashboard provide a graphical interface that allows users to manage storage backends and visualize the metrics and health.
 
 ### Delfin Features
+
 - [Register a Storage Device](#register-storage-device)
     - [REST Access](#rest-access)
-    - [SSH Access](#ssh-access)
+  - [SSH Access](#ssh-access)
```

Comment:
The indentation level seems to be changed here. Was it done intentionally?

Reply:
No, it is done by extension which follows the standard markdown format and in many places the documentation do not follows the indentation standard and other default practices so it shows warning to those... This maybe is resolved while editing..so should i resolve this? as this could also be correct as per indentation guidelines in official markdown.